channels/printer/client/printer_main.c

```diff
@@ -1017,7 +1017,7 @@ printer_DeviceServiceEntry
 	if (!driver)
 	{
 		WLog_ERR(TAG, "Could not get a printer driver!");
-		return CHANNEL_RC_INITIALIZATION_ERROR;
+		goto fail;
```

Comment:
Intentional that this will return `CHANNEL_RC_OK`?

Reply:
no, fixed.